pages/project/delete/index.js

```diff
@@ -0,0 +1,23 @@
+const { Router } = require('express');
+
+module.exports = () => {
+  const app = Router();
+
+  app.use('/draft', (req, res, next) => {
+    if (req.project.status === 'inactive') {
+      return req.api(`/establishment/${req.establishmentId}/project/${req.projectId}`, { method: 'DELETE' })
+        .then(() => res.redirect(`${req.buildRoute('project.list')}?status=inactive`))
+        .catch(next);
+    }
+
+    throw new Error('Active projects cannot be deleted.');
+  });
+
+  app.use('/amendment', (req, res, next) => {
```

Comment:
You don't want `app.use` here because that will listen to all HTTP verbs. You'll want `app.post`.

Reply:
Similar above.